.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Would it be possible to make sure this won't apply to `/ui`?

Reply:
I think the new configuration should give the user the choice when making a PR